ale_linters/go/gometalinter.vim

```diff
@@ -1,8 +1,9 @@
-" Author: Ben Reedy <https://github.com/breed808>
+" Author: Ben Reedy <https://github.com/breed808>, Jeff Willette <jrwillette88@gmail.com>
```

Comment:
Personally I would not change this line. Ben Reedy was the original author who added/created this file. After that several people (including myself) made changes to the file, but that doesn't change the fact that Ben is the original author and we all just made small changes.

Think it will become pretty messy if everyone who touches a file starts adding him/herself as an author...

Reply:
IDK, I'll do whatever @w0rp wants, but I think it becomes pretty clear that he is the original author and others have touched it so that someone may think twice and check the git history before reaching out to him in the future about code that he didn't write.